### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by iannel**

### CODE REVIEW

**Comment by iannel** [comment]
@antoniettaa  @ncaliskan

**Comment by ncaliskan** [comment]
I really liked the fact that you committed a lot to your work which makes it easy to follow up the steps you followed while working on your assignment!! @iannel

File: README.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+
+  * Like *really **really*** steep 
+  
+* It is counterintuitive 
+
+* The error messages are rarely helpful 
+
+R usually makes me want to do this
+
+ ![](https://media.giphy.com/media/3o6ZtiQUuXayJRwwhO/giphy.gif)
```
Comment (ncaliskan) [inline README.md:20]: I love that gif haha and the code seems to work smooth so great job!!!

File: hw_1/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Hi there :wave: My name is Ian Nel and I hate R for the following reasons:
+
+* The learning curve is extremely steep 
+
+  * Like *really **really*** steep 
+  
+* It is counterintuitive 
+
+* The error messages are rarely helpful
```
Comment (ncaliskan) [inline hw_1/README.md:15]: I agree!! :P

**Review by ncaliskan — COMMENTED** [review]
Great job!!

**Pull request merged**